jdaviz/container.vue

```diff
@@ -78,6 +94,41 @@ module.exports = {
        * between a user closing a tab or a re-render. However, when the user closes a tab, the
        * source of the event is a vue component. We can use that distinction as a close signal. */
       source.$root && this.closefn(viewerId);
+    },
+    resizeViewer(viewer) {
```

Comment:
I rotate to 45 degrees and then rotate back, but the Compass zoom box starts acting weird (i.e., it is showing the zoom limits that is a little off from what I am looking at). Since Compass grabs the numbers straight of `viewer.state.x/y_min/max` attributes, something isn't sync-ing right after this operation but I don't know what. Is it the aspect ratio?

Reply:
Or maybe the WCS distortion in the Imviz example notebook data is causing trouble.